packages/dependency-extraction-webpack-plugin/README.md

```diff
@@ -59,7 +81,6 @@ By default, the following module requests are handled:
 | `@babel/runtime/regenerator` | `regeneratorRuntime` | `wp-polyfill` |
 | `@wordpress/*` | `wp['*']` | `wp-*` |
 | `jquery` | `jQuery` | `jquery` |
-| `jquery` | `jQuery` | `jquery` |
```

Comment:
I noticed duplicated line in README and included in this PR as well to avoid opening another PR.